src/other/helloworld/demo/bonjour.js

```diff
@@ -17,6 +17,7 @@ var nomComposant = "wb-bonjour",
 	selecteur = "." + nomComposant,
 	initEvent = "wb-init" + selecteur,
 	$document = wb.doc,
+	defaut = {},
```

Comment:
Knit picking but this is misspelt. Can you also make it plurals for consistency? `defaults`

I notice that some are plural and some ar singular but all seem to be missing the L